### ISSUE TITLE
Stderr: VBoxManage: error: The virtual machine 'saltstates_default_1539870640620_6152' has terminated unexpectedly during startup with exit code 1 (0x1)

### ISSUE THREAD

**Issue by jeffPOG** [hashicorp/vagrant#10314]
I'm currently running vargant version 2.1.5 on my mac high sierra 10.13.6

So after completing the `vagrant init` and `vagrant up` command I'm suppose to have a fully running virtual machine in virtualbox. But after running `vagrant up` I get this instead. 

```
`There was an error while executing `VBoxManage`, a CLI used by Vagrant
for controlling VirtualBox. The command and stderr is shown below.

Command: ["startvm", "3574f45d-aade-4053-af8b-0c88330e855f", "--type", "headless"]

Stderr: VBoxManage: error: The virtual machine 'saltstates_default_1539870640620_6152' has terminated unexpectedly during startup with exit code 1 (0x1)
VBoxManage: error: Details: code NS_ERROR_FAILURE (0x80004005), component MachineWrap, interface IMachine` 

```
Any idea what this is about?

**Comment by chrisroberts** [comment]
There is a log file in the VM directory (usually at `~/VirtualBox VMs/VM_NAME`) which will likely provide extra information on what error was encountered. The path will likely be: `~/VirtualBox VMs/saltstates_default_1539870640620_6152/Logs/VBox.log`. If you do not see any clear reason from the log why the VM failed, feel free to put it in a gist and provide the link here. I'll take a look and see what I can find.

**Comment by jeffPOG** [comment]
@chrisroberts 

I don't see a log file in `~/VirtualBox VMs` but I created a gist of what I do have in there besides the vmdk file. 

https://gist.github.com/jeffPOG/61662d6017048cbba0d963d511bd9466

https://gist.github.com/jeffPOG/d4251dbdbf75ce748077cc6edf9d2a15

I'm not sure if that helps, but if there another location that the logs can be then please let me know.

**Comment by chrisroberts** [comment]
If no log file exists, then your best bet would be to open up the VirtualBox GUI and attempt to start the guest there. If it errors out from there, it should provide some context on the error.

**Comment by reilue** [comment]
Hello there.
Got the exact issue over here. And there is no log file in ~/VirtualBox VMs.
So i take pictures from the VirtualBox GUI.

![screenshot from 2018-10-25 09-51-27](https://user-images.githubusercontent.com/12874996/47506934-22039700-d83f-11e8-92a9-8989a9604492.png)
![screenshot from 2018-10-25 09-50-02](https://user-images.githubusercontent.com/12874996/47506935-22039700-d83f-11e8-95c8-ab76e158a318.png)

**Comment by reilue** [comment]
Hello there.
Got the exact issue over here. And there is no log file in ~/VirtualBox VMs.
So i take pictures from the VirtualBox GUI.

![screenshot from 2018-10-25 09-50-02](https://user-images.githubusercontent.com/12874996/47506975-3e9fcf00-d83f-11e8-9d13-e56bc0aede0a.png)

![screenshot from 2018-10-25 09-51-27](https://user-images.githubusercontent.com/12874996/47506988-43fd1980-d83f-11e8-95ed-06a019351cea.png)

**Comment by reilue** [comment]
I know the issue is close, but this is to help other developers.
Finally it work. After installing VirtualBox 5.1 in ElementaryOS 16.04  
So, in conclusion, pay atention to which **VirtualBox version** can smoothly be run in your **distro**.
In my case, installing VirtualBox 5.1 from **App Center** do the trick. in other distros, **SoftwareCenter** (or similars) might help you. 
Nailed it & share.

**Comment by jeffPOG** [comment]
So I figured it out. Mac High Sierra prevents kernel chances after 30 mins of installing an application in the system or security preferences. So I had to uninstall virtual box then reinstall it and then accept all system changes in security changes in the system preferences after running `vagrant up`

**Comment by gutitrombotto** [comment]
In my case, the error was that I was using Virtualbox 6.0.8 and Vagrant box 'laravel / homestead' version '7.2.1'. I was having the  exact same error.
I solved it uninstalling Virtualbox 6.0.8 and install Virtualbox 5.2.18 using apt-get install virtualbox instead of installing it throgh the official webpage download
